### ISSUE TITLE
Adding Jquery 

### ISSUE THREAD

**Issue by bdpettway19** [bdpettway19/MI-449-SS17-741-css-frameworks-p1twXJ#1]
I'm having an issue adding the JQuery element to my webpages. Can you help me figure out what I'm missing?

**Comment by bdpettway19** [comment]
One of the main problems is the inability to use the dropdown option. When ever I scroll over the LANDSCAPES option nothing happens. I also feel some code may be in the wrong order, but I've matched the code w/ the code shown on  bootstrap.

**Comment by bdpettway19** [comment]
The dropdown is working on my latest code, but inside the web developer inspector it still says Error: Bootstrap's Javascript requires Jquery.

**Comment by stuartpearman** [comment]
Hey Barry, you have 2 links to bootstrap.js. One on line 13 and one on line 53, after you link to the jquery library. Delete the first link, and you should stop seeing that error.

**Comment by stuartpearman** [comment]
Also, right now it's setup so that you'll have to actually click the "Landscapes" tab to see the dropdown.

**Comment by bdpettway19** [comment]
Done

**Comment by stuartpearman** [comment]
Can you post the link to your hosted site?

**Comment by bdpettway19** [comment]
https://bdpettway19.github.io/MI-449-SS17-741-css-frameworks-p1twXJ/

**Comment by stuartpearman** [comment]
Thanks Barry! Katie told you not to worry about the links in the navbar, right?

**Comment by bdpettway19** [comment]
Yes, she told me they would become useful once I got further into the lessons. Thanks!
 

    On Wednesday, January 25, 2017 4:12 PM, Stuart Pearman <notifications@github.com> wrote:
 

 Thanks Barry! Katie told you not to worry about the links in the navbar, right?—

**Comment by stuartpearman** [comment]
Ok cool! So here are a few things I would change before moving on:

1. For your images, instead of setting a width and height to a pixel amount, set them to a 100% width (no height)
2. The classes you use for columns `col-xs-12 col-md-6 col-sm-12 col-lg-6` can actually be replaced with `col-md-6`. This will do the same thing, but using 1 class instead of 4.

Let me know when you've made changes 😎

**Comment by bdpettway19** [comment]
Thanks Stuart! I've gone in and made the changes.

**Comment by stuartpearman** [comment]
Hey Barry, it doesn't look like any changes have been made. Did you push to GitHub?

**Comment by bdpettway19** [comment]
That's exactly what I forgot to do. I've made the following corrections.

**Comment by stuartpearman** [comment]
Nice, looks good Barry! :shipit:
